packages/backend-function/API.md

```diff
@@ -21,6 +21,9 @@ export type CronSchedule = `${string} ${string} ${string} ${string} ${string}` |
 // @public
 export const defineFunction: (props?: FunctionProps) => ConstructFactory<ResourceProvider<FunctionResources> & ResourceAccessAcceptorFactory & AddEnvironmentFactory>;
 
+// @public (undocumented)
+export type FunctionLayerReferences = Record<string, string>;

```

Comment:
inlined this. But to note there is a way we can add a template string literal this but would require a workaround wasnt comfortable using this as unsure if it will be removed: https://github.com/Microsoft/TypeScript/issues/29729#issuecomment-567871939

https://www.typescriptlang.org/play/?#code/C4TwDgpgBAYghgGwQIzgYwNYGVgCcCWAdgOZQC8UAznkaQGRQDeUcuhAMvhhAPwBcLQiCgBfANwBYAFChIsAK6E0wfAHsOcEBFwBBNuSgADVoT5wA7pT4I4AW2QATOHwAkjagRIjX7ml+ua2j4etN5uIV6GUAA+sIgo6Nh+xJIy4NAwispqGlq6bADyBMREiAbGbGaWAfZOwclhvp7E3jZ59c2NES2GqdJo6tRQwBDUAplKKursgfmERfglhGUUAOQmVVY2tc4jY3vAAIwB7QcATHwHAMyrqQOEQ+fjWVO52nqEBuuVFlt2jrtRsBLkDjm0gk9rrdpGBcKo0KNKAA6CCEABuSIOAFF0QYDndBsBhkCrs9JjkZnkPgslisoLD4YiUejMUCcWiWJQqMkCQ8iQcACxk7LTWYfAwMhGUZGojHY3FwLndPpSaTSIA